Ahorn/entities/connectedBlocks/ConnectedTempleCrackedBlock.jl

```diff
@@ -0,0 +1,70 @@
+module CommunalHelperConnectedTempleCrackedBlock
+
+using ..Ahorn, Maple
+
+@mapdef Entity "CommunalHelper/ConnectedTempleCrackedBlock" ConnectedTempleCrackedBlock(
+    x::Integer,
+    y::Integer,
+    width::Integer=Maple.defaultBlockWidth,
+    height::Integer=Maple.defaultBlockWidth,
+    persistent::Bool=false
+)
+
+const placements = Ahorn.PlacementDict(
+    "Connected Temple Cracked Block (CommunalHelper)" => Ahorn.EntityPlacement(
+       ConnectedTempleCrackedBlock,
+        "rectangle",
+        Dict{String, Any}(
+            "persistent" => false
+        )
+    ),
+)
+
+Ahorn.minimumSize(entity::ConnectedTempleCrackedBlock) = 16, 16
+Ahorn.resizable(entity::ConnectedTempleCrackedBlock) = true, true
+
+Ahorn.selection(entity::ConnectedTempleCrackedBlock) = Ahorn.getEntityRectangle(entity)
+
+frame = "objects/temple/breakBlock00"
+
+function rendertempleCrackedBlock(ctx::Ahorn.Cairo.CairoContext, x::Number, y::Number, width::Number, height::Number)
+    tilesWidth = div(width, 8)
+    tilesHeight = div(height, 8)
+
+    Ahorn.Cairo.save(ctx)
+
+    Ahorn.rectangle(ctx, 0, 0, width, height)
+    Ahorn.clip(ctx)
+
+    for i in 0:ceil(Int, tilesWidth / 4)
+        Ahorn.drawImage(ctx, frame, i * 32 + 8, 0, 8, 0, 32, 8)
+
+        for j in 0:ceil(Int, tilesHeight / 4)
+            Ahorn.drawImage(ctx, frame, i * 32 + 8, j * 32 + 8, 8, 8, 32, 32)
+
+            Ahorn.drawImage(ctx, frame, 0, j * 32 + 8, 0, 8, 8, 32)
+            Ahorn.drawImage(ctx, frame, width - 8, j * 32 + 8, 40, 8, 8, 32)
+        end
+
+        Ahorn.drawImage(ctx, frame, i * 32 + 8, height - 8, 8, 40, 32, 8)
+    end
+
+    Ahorn.drawImage(ctx, frame, 0, 0, 0, 0, 8, 8)
+    Ahorn.drawImage(ctx, frame, width - 8, 0, 40, 0, 8, 8)
+    Ahorn.drawImage(ctx, frame, 0, height - 8, 0, 40, 8, 8)
+    Ahorn.drawImage(ctx, frame, width - 8, height - 8, 40, 40, 8, 8)
+
+    Ahorn.restore(ctx)
+end
+
+function Ahorn.render(ctx::Ahorn.Cairo.CairoContext, entity::ConnectedTempleCrackedBlock, room::Maple.Room)
+    x = Int(get(entity.data, "x", 0))
+    y = Int(get(entity.data, "y", 0))
+
+    width = Int(get(entity.data, "width", 32))
+    height = Int(get(entity.data, "height", 32))
+
+    rendertempleCrackedBlock(ctx, x, y, width, height)
```

Comment:
You did take care of the Lönn plugin, which is awesome, but you forgot to update the rendering logic for the Ahorn plugin. It would be awesome if the two plugins could display similarly in both Ahorn and Lönn.
You can take a look at the [ConnectedZipMover.jl](https://github.com/CommunalHelper/CommunalHelper/blob/dev/Ahorn/entities/connectedBlocks/ConnectedZipMover.jl) plugin to see how to do the same, but in Julia for Ahorn.

_(In reality though, this isn't that much of a issue, since we're not really supporting Ahorn anymore)._

Reply:
I'm not sure what you mean here. They appear almost identically in loenn + ahorn for me, with only a minor tiling difference where it meets extensions that doesnt seem important to me since we're phasing out ahorn anyway.
Is there another problem that I'm overlooking?

Ahorn:
![screenshot-1 png (139)](https://user-images.githubusercontent.com/109830893/189544579-5c810e47-3ca2-4885-8fac-974a7f3252dd.png)

Loenn:
![screenshot-1 png (140)](https://user-images.githubusercontent.com/109830893/189544607-de0d51cf-56b7-4f0b-93f7-e26e4c1f1798.png)

Reply:
Yeah, that's fine. It's not as bad as I thought it would be.